zephyr/Kconfig

```diff
@@ -33,6 +33,16 @@ config MCUBOOT_BUILD_STRATEGY_FROM_SOURCE
 
 endchoice
 
+config MCUBOOT_VERSION
+        string "MCUbot version"
+        default "0.0.1"
+	help
+	  Value to be passed as 'version' argument to 'imgtool.py'.
+	  Note that no semantics are connected to this variable.
+	  It does not provide downgrade prevention, and is only valuable for
+	  debugging purposes. Format: maj.min.rev+build with latter parts
+	  optional.
```

Comment:
Can we make it clearer that MCUBOOT_VERSION is not related to the MCUBoot version but the version of the image?

Reply:
e.g. MCUBOOT_IMAGE_VERSION